src/dev/performance/run_performance_cli.ts

```diff
@@ -103,7 +103,7 @@ run(
       process.stdout.write(`--- Starting ES\n`);
       await procRunner.run('es', {
         cmd: 'node',
-        args: ['scripts/es', 'snapshot'],
+        args: ['scripts/es', 'snapshot', '--license=trial'],
```

Comment:
This change will affect all the journeys and don't we want it. Do you might passing licence with a flag?

Reply:
Did you try to pass it within your config file?

Reply:
```
      esTestCluster: {
        ...xPackApiIntegrationTestsConfig.get('esTestCluster'),
        license: trial,
        ssl,
        serverArgs: [
          `xpack.license.self_generated.type=${license}`,
          `xpack.security.enabled=${!disabledPlugins.includes('security')}`,
        ],
```

Reply:
I see, I will have a look how we can make it work based on config settings

Reply:
We discussed it internally and think that using `trial` for all journeys is the reasonable approach. Just to check if it will affect performance metrics, I will schedule few runs for existing journeys and review it.